core/scripts/chaincli/handler/keeper.go

```diff
@@ -106,15 +91,15 @@ func (h *Keeper) deployUpkeeps(ctx context.Context, registryInstance *keeper.Kee
 		if err != nil {
 			log.Fatal(i, ": DeployAbi failed - ", err)
 		}
-		waitDeployment(ctx, h.client, deployUpkeepTx)
+		h.waitDeployment(ctx, deployUpkeepTx)
 		log.Println(i, upkeepAddr.Hex(), ": Upkeep deployed - ", deployUpkeepTx.Hash().Hex())
 
 		// Approve
 		approveUpkeepTx, err := h.linkToken.Approve(h.buildTxOpts(ctx), upkeepAddr, h.approveAmount)
```

Comment:
figured out why the upkeeps were failing funding. upkeepAddr is not the sender it needs to be the registry address.

Reply:
fixed